ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -211,8 +197,8 @@ tests:
         pod should start\| pod sysctls\| build volumes should mount given secrets
         and configmaps into the build pod
     workflow: openshift-e2e-aws-heterogeneous
-- as: ocp-e2e-aws-ovn-heterogeneous-day-0
-  interval: 24h
+- as: ocp-e2e-aws-ovn-multi-day-0-x-a
```

Comment:
can we skip day0 aws/gcp jobs for now? We need a green signal to have a chance to get GA in 4.17.

Reply:
Changes are removed.